indico/MaKaC/webinterface/pages/conferences.py

```diff
@@ -1010,175 +1012,23 @@ def _getNavigationDrawer(self):
-
-        canModify = self._conf.canModify(self._rh.getAW())
-        isReviewingStaff = RCReviewingStaff.hasRights(self._rh)
-        isPRM = RCPaperReviewManager.hasRights(self._rh)
-        #isAM = RCAbstractManager.hasRights(self._rh)
-        isRegistrar = self._conf.canManageRegistration(self._rh.getAW().getUser())
-
-        if not canModify:
-            self._generalSettingsMenuItem.setVisible(False)
-            self._timetableMenuItem.setVisible(False)
-            self._programMenuItem.setVisible(False)
-            self._participantsMenuItem.setVisible(False)
-            self._listingsMenuItem.setVisible(False)
-            self._ACMenuItem.setVisible(False)
-            self._toolsMenuItem.setVisible(False)
-
-        if not (Config.getInstance().getIsRoomBookingActive() and canModify):
-            self._roomBookingMenuItem.setVisible(False)
-
-        #if not (self._conf.hasEnabledSection("cfa") and (canModify or isAM)):
-        if not (self._conf.hasEnabledSection("cfa") and (canModify)):
-            self._abstractMenuItem.setVisible(False)
-
-        if not (canModify or isPRM):
-            self._contribListMenuItem.setVisible(False)
-
-        if not (self._conf.hasEnabledSection("regForm") and (canModify or isRegistrar)):
-            self._regFormMenuItem.setVisible(False)
-
-        if not (self._conf.getType() == "conference" and (canModify or isReviewingStaff)):
-            self._reviewingMenuItem.setVisible(False)
-        else: #reviewing tab is enabled
-            if isReviewingStaff and not canModify:
-                self._reviewingMenuItem.setVisible(True)
-        # For now we don't want the paper reviewing to be displayed
-        #self._reviewingMenuItem.setVisible(False)
-
-        #we hide the Advanced Options section if it has no items
-        if not self._advancedOptionsSection.hasVisibleItems():
-            self._advancedOptionsSection.setVisible(False)
-
-        # we disable the Participants section for events of type conference
-        if self._conf.getType() == 'conference':
-            self._participantsMenuItem.setVisible(False)
+    def _applyFrame(self, body):
+        frame = wcomponents.WConferenceModifFrame(self._conf, self._getAW())
 
-        wf = self._rh.getWebFactory()
-        if wf:
-            wf.customiseSideMenu( self )
-
-    def _setActiveSideMenuItem( self ):
-        pass
-
-    def _applyFrame( self, body ):
-        frame = wcomponents.WConferenceModifFrame( self._conf, self._getAW())
-
-        sideMenu = self._sideMenu.getHTML()
+        params = {
+            "categDisplayURLGen": urlHandlers.UHCategoryDisplay.getURL,
+            "confDisplayURLGen": urlHandlers.UHConferenceDisplay.getURL,
+            "event": "Conference",
+            "sideMenu": render_sidemenu('event-management-sidemenu', active_item=self.sidemenu_option, old_style=True,
+                                        event=self._conf)
```

Comment:
We could already use `self._conf.as_event` here and adapt the signals accordingly (`event.can_manage(session.user, allow_key=True)` for items that require full management access)